tests/TestCase/Routing/RouteBuilderTest.php

```diff
@@ -730,42 +577,6 @@ public function testResourcesInScope(): void
         $this->assertSame('/api/articles/99.json', $url);
     }
 
-    /**
-     * Test resource parsing.
-     */
-    public function testResourcesParsing(): void
-    {
-        $routes = new RouteBuilder($this->collection, '/');
-        $routes->resources('Articles');
-
-        $this->deprecated(function () {
-            $result = $this->collection->parse('/articles', 'GET');
```

Comment:
Should we update these to use parseRequest instead? I didn't see tests covering resource route parsing.